src/helm/values.yaml

```
@@ -0,0 +1,110 @@
+joanie_nginx_status_endpoint: "/__status__"
+joanie_nginx_admin_ip_whitelist: []
+joanie_nginx_bypass_htaccess_ip_whitelist: []
+joanie_nginx_static_cache_expires: "1M"
+http_basic_auth_user_file: "/etc/nginx/conf/htpasswd"
+http_basic_auth_message: "Restricted Area"
+
+# -- postgresql
+joanie_postgresql_version: "9.6"
+joanie_database_engine: "django.db.backends.postgresql_psycopg2"
+joanie_database_image_name: "postgres"
+joanie_database_image_tag: "12"
+# The image pull secret name should match the name of your secret created to
+# login to your private docker registry
+joanie_database_image_pull_secret_name: ""
+joanie_database_host: "postgresql"
+joanie_database_port: 5432
+joanie_database_name: "joanie"
+endpoint_postgresql_ip: ""
+
+# -- postgresql secrets
+DB_USER: fun
+DB_PASSWORD: pass
+
+# -- joanie
+joanie_image_name: "fundocker/joanie"
+joanie_image_tag: "main"
+# The image pull secret name should match the name of your secret created to
+# login to your private docker registry
+joanie_image_pull_secret_name: ""
+joanie_django_port: 8000
+joanie_app_replicas: 1
+joanie_django_settings_module: "joanie.settings"
+joanie_django_configuration: "Development"
+joanie_database_secret_name: |-
+  "joanie-{{ default "undefined-joanie-vault-checksum" .Values.joanie_vault_checksum }}"
+joanie_activate_http_basic_auth: false
+joanie_secret_name: |-
+  "joanie-postgresql-{{ default "undefined-joanie-vault-checksum" .Values.joanie_vault_checksum }}"
+
+# -- joanie secrets
+DJANGO_SECRET_KEY: ThisIsAnExampleKeyForTestPurposeOnly
+# FIXME uncomment this variable and replace with your sentry's credentials
+# DJANGO_SENTRY_DSN: https://super:django@sentry.io/foo
+DJANGO_JWT_PRIVATE_SIGNING_KEY: 1234
+
+# -- resources
+app_resources:
+  requests:
+    cpu: "50m"
+    memory: "500Mi"
+joanie_app_resources: "{{ toYaml .Values.app_resources  }}"
+joanie_app_job_db_migrate_resources: "{{ toYaml .Values.app_resources   }}"
+
+joanie_nginx_resources:
+  requests:
+    cpu: "10m"
+    memory: "5Mi"
+
+joanie_database_resources:
+  requests:
+    cpu: "10m"
+    memory: "100Mi"
+
+databases:
+  engine: "postgresql"
+  release: "9.6"
+
+# -- general settings
+container_uid: 10000
+container_gid: 0
+customer: "ci"
+# TO BE REMOVED (blue green deployment will no longer be done using the Arnold project)
+prefix: "current"
+trashable_env_types: ["ci", "development", "feature"]
+blue_green_prefixes: ["previous", "current", "next"]
+env_type: "ci"
+acme_env: "staging"
+acme_enabled_route_prefix: []
+
+# docker registries credentials used to create the imagePull secrets define as many docker
+# registries as needed
+dockerRegistries:
+  - firstDockerRegistry:
+    name: registryname1
+    registry: registryserver1
+    email: email
+    username: registryusername
+    password: registrypassword
+  - secondDockerRegistry:
+    name: registryname2
+    registry: registryserver2
+    email: email
+    username: registryusername
+    password: registrypassword
```

As this is a list of objects, you don't need to add a key:

```suggestion
  - name: registryname1
    registry: registryserver1
    email: email
    username: registryusername
    password: registrypassword
  - name: registryname2
    registry: registryserver2
    email: email
    username: registryusername
    password: registrypassword
```